.gitignore

```diff
@@ -15,3 +15,6 @@ CIScripts/JenkinsTesterDSC/JenkinsTester
 
 ### Python ###
 *.pyc
+
+### Visual Studio Code
+.vscode/
```

Comment:
Why was this added?

Reply:
Why not? :D Most of the team uses vscode